index.html

```diff
@@ -0,0 +1,231 @@
+<div class="page1">
+<div class="page1__header">
+    <ul class="page1__links">
+        <li class="page1__big-link">Medical core</li>
+        <li class="page1__sub-links page1__sub-links--further">Home</li>
+        <li class="page1__sub-links">Product</li>
+        <li class="page1__sub-links">Pricing</li>
+        <li class="page1__sub-links page1__sub-links--far">Contact</li>
+        <li class="page1__link-apart"><a class="page1__sub-links page1__sub-links--green" href="">Login</a></li>
+         <li ><button class="btn btn--with-icon">JOIN US
+          <svg style=" margin-left: 1rem;" width="12" height="10" viewBox="0 0 12 10" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M0 5C0 4.81059 0.079009 4.62895 0.219646 4.49502C0.360282 4.36109 0.551026 4.28584 0.749916 4.28584H9.43845L6.21831 1.22068C6.07749 1.08658 5.99838 0.904705 5.99838 0.715059C5.99838 0.525414 6.07749 0.343536 6.21831 0.209436C6.35912 0.0753365 6.5501 0 6.74925 0C6.94839 0 7.13937 0.0753365 7.28019 0.209436L11.7797 4.49438C11.8495 4.56072 11.9049 4.63952 11.9427 4.72629C11.9805 4.81305 12 4.90606 12 5C12 5.09394 11.9805 5.18695 11.9427 5.27371C11.9049 5.36048 11.8495 5.43928 11.7797 5.50562L7.28019 9.79056C7.13937 9.92466 6.94839 10 6.74925 10C6.5501 10 6.35912 9.92466 6.21831 9.79056C6.07749 9.65646 5.99838 9.47459 5.99838 9.28494C5.99838 9.0953 6.07749 8.91342 6.21831 8.77932L9.43845 5.71416H0.749916C0.551026 5.71416 0.360282 5.63892 0.219646 5.50499C0.079009 5.37106 0 5.18941 0 5Z" fill="white"/>
+</svg>
+</button></li>
+
+    </ul>
+</div>
+<div class="page1__words">
+  <h5 class="page1__headings page1__headings--short">Welcome</h5>
+  <h1 class="page1__headings page1__headings--larg">A Great Place to Recieve Care</h1>
+  <h4 class="page1__headings page1__headings--small">Get your best looking smile now!</h4>
+  <div class="page1__buttons-end">
+  <button class="btn btn--no-icon">Join Us</button>
+          <button class="page1__btn">Learn More</button> 
+</div>
+</div>
+</div>
+
+
+
+<div class="page2">
+    <img class="page2__image" src="./assets/doct.svg" alt="">
+    <div class="page2__body">
+        <img src="./assets/doct2.svg" alt="">
+      <div class="page2__sub-body">
+
+    <h2 class="heading6 heading6--page2">Meet Our Experts</h2>
+    <p class="paragraph paragraph--page2">Problems trying to resolve the conflict between the two major realms of classical physics: Newtorian mechanics</p>
+          <a href="" class="link">
+          </div>
+      <h6 class="heading6 heading6--page2">Learn More 
+       <img class="icon-flesh" src="./assets/flesh.svg" alt="">
+      </h6>
+    </a>
+      </div>
+   
+
+  </div>
+
+
+
+
+  <div class="page3">
+    <h6 class="heading6 heading6--page3">Team</h6>
+    <h3 class="heading3">Our Team</h3>
+    <p class="paragraph paragraph--page3">Problems trying to resolve the conflict between the two major realms of classical physics: Newtorian mechanics</p>
```

Comment:
**paragraph** is an element not a block. where does **paragraph** element belong to.